editoast/src/views/paced_train.rs

```diff
@@ -315,19 +399,35 @@ async fn project_path(
 
 #[cfg(test)]
 mod tests {
-    use axum::http::StatusCode;
-    use pretty_assertions::assert_eq;
+    use std::collections::HashMap;
+
+    use chrono::Duration;
+    use editoast_models::DbConnectionPoolV2;
+    use editoast_schemas::paced_train::{Paced, PacedTrainBase};
+    use editoast_schemas::train_schedule::TrainScheduleBase;
     use rstest::rstest;
     use serde_json::json;
 
-    use crate::error::InternalError;
-    use crate::models::fixtures::create_simple_paced_train;
-    use crate::models::fixtures::create_timetable;
-    use crate::models::fixtures::simple_paced_train_base;
-    use crate::models::paced_train::PacedTrain;
+    use crate::core::simulation::{
+        CompleteReportTrain, ElectricalProfiles, ReportTrain, SimulationResponse,
+        SpeedLimitProperties,
+    };
+    use crate::models::fixtures::create_fast_rolling_stock;
+    use crate::models::fixtures::create_small_infra;
+    use crate::models::paced_train::PacedTrainChangeset;
     use crate::models::prelude::*;
-    use crate::views::paced_train::PacedTrainResult;
-    use crate::views::test_app::TestAppBuilder;
+    use crate::views::test_app::TestApp;
+    use crate::views::tests::mocked_core_pathfinding_sim_and_proj;
+    use crate::views::InternalError;
+    use crate::views::SimulationSummaryResult;
+    use crate::{
+        models::{
+            fixtures::{create_simple_paced_train, create_timetable, simple_paced_train_base},
+            paced_train::PacedTrain,
+        },
+        views::{paced_train::PacedTrainResult, test_app::TestAppBuilder},
+    };
```

Comment:
nitpick: can you expand all of `use` (one line for each). It's easier to review diff when they're flatten. You might be able to configure `rustfmt` to do that automatically (need the nightly formatter though).